### PR TITLE
Updated for build scan plugin v2.0

### PR DESCRIPTION
**Pull request by timyates**
### Context

2.0 of the scan plugin is available for Gradle 5.0

### CODE REVIEW

File: subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy [hunk 1]
```diff
@@ -52,7 +52,8 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
         "1.14",
         "1.15.1",
         "1.15.2",
-        "1.16"
+        "1.16",
+        "2.0"
```
Comment (wolfs) [inline subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy]: I guess the `@Ignore` from this test should be removed. I also think that only `2.0` is supported and no other version.

Comment (timyates) [reply subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy]: Fixed, I removed the `GRACEFULLY_UNSUPPORTED` list and test, as we now seem to print a warning for any version prior to 2.0

**Comment by timyates** [comment]
Added the tests back in, and fixed up where we were referring to `licenseAgreementUrl` and `licenseAgreementAgree` as they've now gone in favor of the new properties

🤞

@wolfs @facewindu Can you check I've done good? 😉

**Comment by facewindu** [comment]
good to be merged

**Comment by timyates** [comment]
Merged and merged `release` into `master`

Thanks all 👍

**Pull request merged**